src/test/scala/com/databricks/labs/mosaic/expressions/geometry/ST_UnaryUnionTest.scala

```diff
@@ -1,41 +1,13 @@
 package com.databricks.labs.mosaic.expressions.geometry
 
-import com.databricks.labs.mosaic.core.geometry.api.GeometryAPI.{ESRI, JTS}
-import com.databricks.labs.mosaic.core.index.{BNGIndexSystem, H3IndexSystem}
-import org.apache.spark.sql.QueryTest
-import org.apache.spark.sql.catalyst.expressions.CodegenObjectFactoryMode
-import org.apache.spark.sql.internal.SQLConf
+import com.databricks.labs.mosaic.test.MosaicSpatialQueryTest
 import org.apache.spark.sql.test.SharedSparkSession
 
-class ST_UnaryUnionTest extends QueryTest with SharedSparkSession with ST_UnaryUnionBehaviours {
-    private val noCodegen =
-        withSQLConf(
-          SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "false",
-          SQLConf.CODEGEN_FACTORY_MODE.key -> CodegenObjectFactoryMode.NO_CODEGEN.toString
-        ) _
+class ST_UnaryUnionTest extends MosaicSpatialQueryTest with SharedSparkSession with ST_UnaryUnionBehaviours {
 
-    private val codegenOnly =
-        withSQLConf(
```

Comment:
what happened with this section?
I never got any code generation to properly execute without using withSQLConf blocks.
Had massive issues with false passing tests and they'd fail on a cluster with exact same code.

Reply:
found it!